docs/learning/ECSQLTutorial/ClassFilter.md

```diff
@@ -46,8 +44,6 @@ ECSQL allows filters by class
 > SELECT * FROM bis.Element WHERE ECClassId IS NOT (ONLY Generic.PhysicalObject, ONLY BisCore.LightLocation)
 > ```
 
-<iframe class="embedded-console" src="https://imodelconsole.bentley.com/?embedded=true&nosignin=true&imodel=HouseSample&query=SELECT * FROM bis.Element WHERE ECClassId IS NOT (ONLY Generic.PhysicalObject, ONLY Biscore.LightLocation)"></iframe>
-
 ---
 
-[**< Previous**](./ChangeSummaryQueries.md) &nbsp; | &nbsp; [**Next >**](./ConditionalExpr.md)
+[**< Previous**](./ChangeSummaryQueries.md) &nbsp;
```

Comment:
Not a huge fan of the bold links:
<img width="176" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/78756012/571548e6-a6f0-435a-abae-32491fe5e6c8">

Looks fine without and using unicon chevrons `‹` and `›` I think
<img width="176" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/78756012/ad8433da-7d98-44a0-bb64-cf7753568c49">